src/mainloop.cpp

```diff
@@ -228,7 +228,7 @@ void Mainloop::handle_tcp_connection()
     log_debug("TCP Server: New client");
 
     auto *tcp = new TcpEndpoint{"dynamic"};
-    tcp->retry_timeout = 0; // disable retry on dynamic endpoints by the TCP server
+    tcp->set_retry_timeout(0); // disable retry on dynamic endpoints by the TCP server
```

Comment:
this can only be set be the configuration, which is now handled in the ::setup() method. So maybe we should just remove this line

Reply:
Since the method will not change the underlying setting (timeout handling) at any place in time, I removed the method all together and incorporated the setting to zero into `accept()`.